src/ToggleButton/tests.jsx

```diff
@@ -0,0 +1,202 @@
+/* eslint-env node, mocha */
+/* global expect */
+/* eslint no-console: 0*/
+/* eslint-disable no-magic-numbers, no-multi-str, no-unused-expressions */
+
+
+import React               from 'react';
+import { shallow, mount }  from 'enzyme';
+
+import Icon                from '../Icon';
+
+import ToggleButton        from './index';
+
+
+describe( 'ToggleButton', () =>
+{
+    let instance;
+    let wrapper;
+
+    beforeEach( () =>
+    {
+        wrapper = shallow( <ToggleButton /> );
+        instance = wrapper.instance();
+    } );
+    describe( 'label', () =>
+    {
+        it( 'should be undefined by "default"', () =>
+        {
+            expect( instance.props.id ).to.be.undefined;
+        } );
+    } );
+    describe( 'role', () =>
+    {
+        it( 'should be "primary" by "default"', () =>
+        {
+            expect( instance.props.role ).to.equal( 'primary' );
+        } );
+        it( 'should be "secondary" as well when it passed to', () =>
```

Comment:
This test doesn't really test anything, please remove it. (If I set the prop to `'secondary'` then of course it will equal `'secondary'` :slightly_smiling_face: )